wnfs-wasm/Cargo.toml

```
@@ -12,7 +12,7 @@ categories = [
 license = "Apache-2.0"
 readme = "README.md"
 edition = "2021"
-repository = "https://github.com/WebNativeFileSystem/rs-wnfs/tree/main/crates/wasm"
+repository = "https://github.com/WebNativeFileSystem/rs-wnfs/tree/main/wnfs-wasm"
```

We should also update this to `wnfs-wg`
```suggestion
repository = "https://github.com/wnfs-wg/rs-wnfs/tree/main/wnfs-wasm"
```